units/systemd-hibernate-resume@.service.in

```diff
@@ -14,7 +14,7 @@ DefaultDependencies=no
 BindsTo=%i.device
 Wants=local-fs-pre.target
 After=%i.device
-Before=local-fs-pre.target
+Before=local-fs-pre.target sysroot.mount sysroot-usr.mount
```

Comment:
hmm, I think it would be better to change sysroot-usr.mount to sysusr.mount

(background: we want to support environments where only /usr/ exists initially, and where repart data is stored in it which tells us how to create the root fs. thus we want to mount /usr before the root fs. But that cannot work if we nest /usr/ right-away inside /sysroot/. hence we mount /usr to /sysusr/ first, then do the repart thing, and eventually we bind mount it into /sysroot/usr/. my understanding here is that we shouldn't even mount /sysusr/ at all, hence that's should be what we order against here.)

Reply:
That's what a sync point would be useful for, as explained in my other comments.

Reply:
Let's discuss that separately, naming of that unit is not directly related.